src/app/calculator/waste-water/state-point-analysis/state-point-analysis-form/state-point-analysis-form.component.html

```diff
@@ -63,22 +63,57 @@
+                <span *ngIf="settings.unitsOfMeasure == 'Metric'" class="input-group-addon units">m</span>
+                <span class="alert-danger pull-right small"
+                    *ngIf="form.controls.diameter.invalid && !form.controls.diameter.pristine">
+                    <span *ngIf="form.controls.diameter.errors.min">Value can't be less than
+                        {{form.controls.diameter.errors.min.min}}
+                        <span *ngIf="settings.unitsOfMeasure != 'Metric'">ft</span>
+                        <span *ngIf="settings.unitsOfMeasure == 'Metric'">m</span>.
+                    </span>
+                </span>
+            </div>
+        </div>
+
+        <div class="form-group">
+            <label for="{{'areaOfClarifier_'+idString}}">Clarifier Area
+                <a class="click-link small form-text" (click)="showHideInputField()"
+                    *ngIf="form.controls.userDefinedArea.value">Auto Calculate From Diameter</a>
+                <a class="click-link small form-text" (click)="showHideInputField()"
+                    *ngIf="!form.controls.userDefinedArea.value">Define Clarifier Area</a>
+            </label>
+            <div class="input-group" *ngIf="this.form.controls.userDefinedArea.value">
                 <input type="number" name="areaOfClarifier" class="form-control number-input-field"
                     id="{{'areaOfClarifier_'+idString}}" (focus)="focusField('areaOfClarifier')" (input)="calculate()"
                     formControlName="areaOfClarifier" [readonly]="!selected">
-                    <span *ngIf="settings.unitsOfMeasure == 'Imperial'" class="input-group-addon units">
-                        ft<sup>2</sup></span>
-                    <span *ngIf="settings.unitsOfMeasure != 'Imperial'" class="input-group-addon units">
-                        m<sup>2</sup></span>
+                <span *ngIf="settings.unitsOfMeasure == 'Imperial'" class="input-group-addon units">
+                    ft<sup>2</sup></span>
+                <span *ngIf="settings.unitsOfMeasure != 'Imperial'" class="input-group-addon units">
+                    m<sup>2</sup>
+                </span>
             </div>
             <span *ngIf="form.controls.areaOfClarifier.invalid && !form.controls.areaOfClarifier.pristine">
                 <span class="alert-danger pull-right small" *ngIf="form.controls.areaOfClarifier.errors.required">Value
                     Required</span>
-                <span class="alert-danger pull-right small" *ngIf="form.controls.areaOfClarifier.errors.min">Value can't be negative</span>
+                <span class="alert-danger pull-right small" *ngIf="form.controls.areaOfClarifier.errors.min">Value can't be
+                    negative</span>
             </span>
+            <div class="input-group d-flex justify-content-center small bold" *ngIf="!form.controls.userDefinedArea.value">
```

Comment:
Small enhancement. Let's get an ngIf'd span in here that displays 2 dashes when there is no area value (and still shows units). Search app for "&mdash;" to see usage examples.